addons/website_sale/models/website_snippet_filter.py

```diff
@@ -196,3 +196,10 @@ def _get_products_alternative_products(self, website, limit, domain, context):
                     display_default_code=False,
                 ).search(domain, limit=limit)
         return products
+
+    def _prepare_values(self, limit=None, search_domain=None):
```

Comment:
According to `website/models/website_snippet_filter.py`'s methods order, the override of `_prepare_values` should be the first method of this file.

Reply:
What do you mean?

Reply:
`_prepare_values` is one of the first methods of the file in `website`, it should be the same here in `website_sale`.